gluten-core/src/main/scala/io/glutenproject/GlutenPlugin.scala

```diff
@@ -180,6 +180,15 @@ private[glutenproject] class GlutenDriverPlugin extends DriverPlugin with Loggin
       conf.set("spark.sql.orc.enableVectorizedReader", "false")
       conf.set("spark.sql.inMemoryColumnarStorage.enableVectorizedReader", "false")
     }
+
+    if (
```

Comment:
would be better if you could add a note here

Reply:
Added, thanks